src/wallet/api/wallet_manager.cpp

```diff
@@ -76,17 +76,42 @@ Wallet *WalletManagerImpl::openWallet(const std::string &path, const std::string
     return wallet;
 }
 
-Wallet *WalletManagerImpl::recoveryWallet(const std::string &path, const std::string &memo, bool testnet, uint64_t restoreHeight)
+Wallet *WalletManagerImpl::recoveryWallet(const std::string &path,
+                                                const std::string &memo,
+                                                bool testnet,
+                                                uint64_t restoreHeight)
+{
```

Comment:
Same comment as above, it doesn't seem necessary to change these above 4 lines.

Reply:
it is not necessary to have line breaks at all. the line is imply 129 characters long. which is a lot. also, some lines have breaks, some don't - we should have some sort of consistency - if it's long, break it.

Reply:
I understand your feeling about some ugliness in the code, and it's OK to fix some formatting issues **as long as you do so while also making actual functional changes**. Changes purely for formatting purposes with no actual functional changes shouldn't be allowed, IMO.

Reply:
i agree here fully.  this function has, in fact, been changed but i am happy to make it ugly again if you insist. same goes for recoverFromKeys() above.

Reply:
In this particular example, the existing line

```c++
Wallet *WalletManagerImpl::recoveryWallet(const std::string &path, const std::string &memo, bool testnet, uint64_t restoreHeight)
```

can be kept intact. Please try to make some reasonable effort in minimizing the number of changed lines.

Reply:
@m2049r please re-read CONTRIBUTING.md.

> Patches should be self contained. A good rule of thumb is to have
one patch per separate issue, feature, or logical change. Also, no
other changes, such as random whitespace changes or reindentation.
Following the code style of the particular chunk of code you're
modifying is encouraged.

This is how all large software projects operate. If you don't understand why today, you may in a year or two.

Reply:
i don't understand why you guys think i have changed unrelated stuff. i have not. and i have no intention to. i have touched like 12 methods (half of them new) - which i tried to format according to the surrounding blocks of code, including their declarations. i have not touched any code outside this scope.

it's not that i have gone and reformatted the indentation of the 5 files i changed (which is what the citation is sensibly trying to prevent).

as requested, i have undone some of the formatting so stuff is ugly and inconsistent again.